ops/main.py

```diff
@@ -174,7 +174,8 @@ def main(charm_class):
         juju_event_name = '{}_action'.format(juju_event_name)
 
     model_backend = ops.model.ModelBackend()
-    setup_root_logging(model_backend)
+    debug = ('JUJU_DEBUG' in os.environ)
```

Comment:
This is fine if it's more clear to you, but parenthesis surrounding the expression are not really needed.

Reply:
So it is, very much, for clarity rather than need. 
```
foo = 'string' more stuff
```
looks like the result is a string to me.